plan/validator.go

```diff
@@ -271,6 +271,16 @@ func (v *validator) checkAlterTableGrammar(stmt *ast.AlterTableStmt) {
 			default:
 				// Nothing to do now.
 			}
+		case ast.AlterTableOption:
+			for _, opt := range spec.Options {
+				switch opt.Tp {
```

Comment:
I don't know if there will be any code about other `TableOption.Tp` in the future, so I use `switch` instead of `if-else`